draft-secure-credential-transfer.md

```diff
@@ -203,9 +203,56 @@ Sender device may terminate the secure credential transfer by deleting the mailb
 ~~~
 {: #stateful-flow-image title="Sample stateful workflow"}
 
+## Provisioning Information Structure
+
+The Provisioning Information is the data transfered via the Relay Server between the Sender device and Receiver device. Each use case defines its own specalized Provisioning Information format, but all formats must at least ahead to the following structure.
+
+| Key    | Type   | Description 
+| ------ | ------ | --- 
+| format | String | The Provisioning Information format that the message follows. This is used by the Sender device and Receiver device to know how to parse the message.
+
+##### Provisioning Information Format
+
+Each Provisioning Information format must have the message structure defined in an external specification.
+
+| Format Type                      | Spec Link | Description 
+| -------------------------------- | --------- | ---
+| digitalwallet.carkey.ccc         | [Link]()  | A digital wallet Provisioning Information for sharing a car key that follows the Car Connectivity Consortium specification.
+| digitalwallet.authorizationToken | [Link]()  | A digital wallet Provisioning Information for sharing a generic pass that relies solely on an authorization token.
+
+~~~
+{
+    "format" : "digitalwallet.carkey.ccc",
+    // Additional use case specific fields
+}
+~~~
+
+### Provisioning Information Encryption
+
+Provisioning Information will be stored on the Relay Server encrypted. The Secret used to encrypt the Provisioning Information should given to the Receiver Device via [Share URL](#Share-URL). The encrypted payload should be a data structure having the following key-value pairs:
```

Comment:
'[' and ']' symbols have sensitive nature in this document and only should be used to specifically refer to some document defined in IETF or local reference. This sentence needs re-formatting to avoid document compilation failure